### PR TITLE
RDBC-460 Migrate tests from Java client to Python client 

### PR DESCRIPTION
**Pull request by poissoncorp**
https://issues.hibernatingrhinos.com/issue/RDBC-460

### CODE REVIEW

File: pyravendb/tools/utils.py [hunk 1]
```diff
@@ -291,7 +292,11 @@ def timedelta_to_str(timedelta_obj):
                 timedelta_str += "{0}.".format(days)
             timedelta_str += "{:02}:{:02}:{:02}".format(hours, minutes, seconds)
             if microseconds > 0:
-                timedelta_str += ".{0}".format(microseconds)
+                timedelta_str += "."
+                order_of_magnitude = log10(microseconds).__trunc__()
```
Comment (gregolsky) [inline pyravendb/tools/utils.py]: wouldn't `math.trunc()` be more "elegant" instead?

File: pyravendb/tools/utils.py [hunk 2]
```diff
@@ -291,7 +292,11 @@ def timedelta_to_str(timedelta_obj):
...
+                if order_of_magnitude <= 5:
+                    timedelta_str += "0" * (5 - order_of_magnitude)
```
Comment (gregolsky) [inline pyravendb/tools/utils.py]: isnt that just padding? https://www.w3schools.com/python/trypython.asp?filename=demo_ref_string_rjust2

File: pyravendb/tools/utils.py (hunk 1 above)
Comment (poissoncorp) [reply pyravendb/tools/utils.py]: done

File: pyravendb/tools/utils.py (hunk 2 above)
Comment (poissoncorp) [reply pyravendb/tools/utils.py]: done

**Pull request merged**